vmaas-go/webapp/controllers/cves.go

```diff
@@ -0,0 +1,45 @@
+package controllers
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/gin-gonic/gin"
+	"github.com/redhatinsights/vmaas-lib/vmaas"
+	"github.com/redhatinsights/vmaas/base/core"
+	"github.com/redhatinsights/vmaas/base/utils"
+)
+
+func CvesHandler(c *gin.Context) {
+	if !isCacheLoaded(c) {
+		return
+	}
+	cves := strings.Split(c.Param("cves"), ",")
```

Comment:
GET handler shows results for only 1 CVE, don't split the string, pass what is in cves